src/components/Dialogs/VectorOverlayDialog/VectorOverlayDialogComponent.tsx

```diff
@@ -0,0 +1,410 @@
+                                stepSize={0.1}
+                                onValueChange={dataSource.vectorOverlayConfig.setColormapContrast}
+                            />
+                        </FormGroup>
+                    </React.Fragment>
+                ) : (
+                    <FormGroup inline={true} label="Color">
+                        <ColorPickerComponent
+                            color={dataSource.vectorOverlayConfig.color}
+                            presetColors={SWATCH_COLORS}
+                            setColor={(color: ColorResult) => dataSource.vectorOverlayConfig.setColor(color.rgb)}
+                            disableAlpha={true}
+                            darkTheme={appStore.darkTheme}
+                        />
+                    </FormGroup>
+                )}
+            </div>
+        );
+
+        return (
+            <DraggableDialogComponent dialogProps={dialogProps} helpType={HelpType.VECTOR_OVERLAY} defaultWidth={VectorOverlayDialogComponent.DefaultWidth} defaultHeight={VectorOverlayDialogComponent.DefaultHeight} enableResizing={true}>
+                <div className={Classes.DIALOG_BODY}>
+                    <FormGroup inline={true} label="Data Source">
+                        <DataSourceSelect
+                            activeItem={dataSource}
+                            onItemSelect={appStore.setActiveFrame}
+                            popoverProps={{minimal: true, position: "bottom"}}
+                            filterable={false}
+                            items={appStore.frames}
+                            itemRenderer={this.renderDataSourceSelectItem}
+                            disabled={appStore.animatorStore.animationActive}
+                        >
+                            <Button text={dataSource.filename} rightIcon="double-caret-vertical" alignText={"right"} disabled={appStore.animatorStore.animationActive} />
+                        </DataSourceSelect>
+                    </FormGroup>
+                    <Tabs defaultSelectedTabId={VectorOverlayDialogTabs.Configuration} renderActiveTabPanelOnly={false}>
+                        <Tab id={VectorOverlayDialogTabs.Configuration} title="Configuration" panel={configPanel} panelClassName="vector-overlay-config-panel" />
+                        <Tab id={VectorOverlayDialogTabs.Styling} title="Styling" panel={stylingPanel} panelClassName="vector-overlay-styling-panel" />
+                    </Tabs>
+                </div>
+                <div className={Classes.DIALOG_FOOTER}>
+                    <div className={Classes.DIALOG_FOOTER_ACTIONS}>
+                        <AnchorButton intent={Intent.WARNING} onClick={this.handleClearOverlay} disabled={!dataSource.vectorOverlayConfig.enabled} text="Clear" />
+                        <AnchorButton intent={Intent.SUCCESS} onClick={this.handleApplyOverlay} disabled={!this.configChanged && dataSource.vectorOverlayConfig.enabled} text="Apply" />
```

Comment:
maybe we could disable Apply button when both angle and intensity source is None